frog_api/views/data.py

```diff
@@ -0,0 +1,163 @@
+from typing import Any, List
+
+from django.db import models
+from frog_api.exceptions import *
+from frog_api.models import Entry, Measure, Project, Version
+from frog_api.serializers import TerseEntrySerializer
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from common import get_category, get_project, get_version
```

Comment:
The very top is standard library, the middle is external modules (though not sure why frog_api is there..maybe because we're importing a submodule of it?), and the last section is local modules. This is how vscode organizes imports when you use their feature